docs/proposals/policy.md

```diff
@@ -0,0 +1,146 @@
+
```

Comment:
Just to clarify, once this gets review here it needs to go upstream and get review as well.  So I wouldn't leave it here very long because any substantive discussion will be lost.  First pass only